### ISSUE TITLE
Changing axis title and plotly tooltips

### ISSUE THREAD

**Issue by yean8mun** [oegedijk/explainerdashboard#142]
Hi,

How do I change the graph axis titles or change the tooltips in the plotly graphs?
Are there any public static methods available from explainerdashboard library?

From the image below, I want to change the axis title.
![image](https://user-images.githubusercontent.com/61852565/132133966-3530b40d-18f6-4e60-9434-7996c65ae388.png)

From the image below, I want to change the tooltip for example:
![image](https://user-images.githubusercontent.com/61852565/132134044-fca73cb1-5edd-4f9f-a187-d7cf8f5b7d98.png)

Appreciate it if you could include some code snippet & examples.
Thanks.

**Comment by oegedijk** [comment]
Hi @yean8mun, 

It's not really possible to change the title of the plots. Although you can do it manually if you use the explainer interactively:

```python
fig = explainer.plot_importances()
fig.update_layout(title="New title")
```

You can adjust the tooltips with the `descriptions` parameter:

```python
feature_descriptions = {
    "Sex": "Gender of passenger",
    "Gender": "Gender of passenger",
    "Deck": "The deck the passenger had their cabin on",
    "PassengerClass": "The class of the ticket: 1st, 2nd or 3rd class",
    "Fare": "The amount of money people paid", 
    "Embarked": "the port where the passenger boarded the Titanic. Either Southampton, Cherbourg or Queenstown",
    "Age": "Age of the passenger",
    "No_of_siblings_plus_spouses_on_board": "The sum of the number of siblings plus the number of spouses on board",
    "No_of_parents_plus_children_on_board" : "The sum of the number of parents plus the number of children on board",
}

X_train, y_train, X_test, y_test = titanic_survive()
train_names, test_names = titanic_names()
model = RandomForestClassifier(n_estimators=50, max_depth=5)
model.fit(X_train, y_train)

explainer = ClassifierExplainer(model, X_test, y_test, 
                                cats=['Deck', 'Embarked',
                                    {'Gender': ['Sex_male', 'Sex_female', 'Sex_nan']}],
                                cats_notencoded={'Embarked': 'Stowaway'}, # defaults to 'NOT_ENCODED'
                                descriptions=feature_descriptions, # adds a table and hover labels to dashboard
                                labels=['Not survived', 'Survived'], # defaults to ['0', '1', etc]
                                idxs = test_names, # defaults to X.index
                                index_name = "Passenger", # defaults to X.index.name
                                target = "Survival", # defaults to y.name
                                )
```

**Comment by jennifer-yean** [comment]
@oegedijk thanks for your reply. 

Tried running the above code but fig is blank?
![image](https://user-images.githubusercontent.com/55101406/134031379-e807472f-5772-4e12-ab82-e4a55d7cf42d.png)

**Comment by jennifer-yean** [comment]
Worked out that it's a plotly renderer issue. Thanks it works

**Comment by yean8mun** [comment]
![image](https://user-images.githubusercontent.com/61852565/134551545-ccbeb157-c2f1-4bfa-905f-061b0d4ba6dd.png)

I have used the example from the [titanic demo](http://titanicexplainer.herokuapp.com/) --> Regression Dashboard using the code below:
```
from sklearn.ensemble import RandomForestRegressor

from explainerdashboard import RegressionExplainer, ExplainerDashboard
from explainerdashboard.datasets import titanic_fare, feature_descriptions

X_train, y_train, X_test, y_test = titanic_fare()
model = RandomForestRegressor(n_estimators=50, max_depth=10).fit(X_train, y_train)

explainer = RegressionExplainer(model, X_test, y_test, 
                                cats=['Sex', 'Deck', 'Embarked'], 
                                descriptions=feature_descriptions,
                                units="$")

ExplainerDashboard(explainer).run()
```
I can see the tooltip, which is good.
However, I want to hide the text shown after the bar (refer to the underlined words in the screenshot above).

Any easy way to fix it? Thanks

**Comment by oegedijk** [comment]
Hi @yean8mun , that's weird actually. I don't see them when I run this locally? 

Have you tried updating `dash` to the latest version?

**Comment by oegedijk** [comment]
closing this for now...
